### PR TITLE
Add 9gag random page

### PR DESCRIPTION
**Pull request by martinusso**

### CODE REVIEW

**Comment by RobertoSchneiders** [comment]
Awesome! 

What if the 9gag command accept a parameter with a optinal 9gag section? like.. `9gag trending` or `9gag hot`

**Comment by martinusso** [comment]
I thought about that. but, yet, I don't found a good way to do this.
I'll put on the roadmap ;)

File: 9gag/9gag.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package gag
+
+import (
+	"net/http"
+
+	"github.com/go-chat-bot/bot"
+)
+
+const (
+	rangomPage = "http://9gag.com/random"
```
Comment (fabioxgn) [inline 9gag/9gag.go:10]: image

File: 9gag/9gag.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+)
+
+func gag(command *bot.Cmd) (string, error) {
+	res, err := http.Get(rangomPage)
+	if err != nil {
+		return "", err
+	}
+	return res.Request.URL.String(), nil
```
Comment (fabioxgn) [inline 9gag/9gag.go:18]: I'll try to make it not follow the redirect, we are interested just in the URL itself, not the contents. This is opening the redirected page.

**Pull request merged**